.github/workflows/lint.yml

```
@@ -0,0 +1,27 @@
+name: C Project Build Check
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)